CONTRIBUTING.md

```diff
@@ -114,6 +114,11 @@ and the rest of contributors.
   the folder. If a component can not be properly tested within the existing framework, it must increase the non testable
   components number with a comment within the PR explaining as to why it can not be tested.
 
+### Releasing New Components
+After a component has been approved and merged, and has been enabled in `internal/components/`, it must be added to the
+[OpenTelemetry Collector distribution repository's manifest.yaml](https://github.com/open-telemetry/opentelemetry-collector-releases/blob/main/distributions/otelcol-contrib/manifest.yaml)
```

Comment:
```suggestion
[OpenTelemetry Collector Contrib's release manifest.yaml](https://github.com/open-telemetry/opentelemetry-collector-releases/blob/main/distributions/otelcol-contrib/manifest.yaml)
```
Not sure if this is better but seems a little clearer to me.

Reply:
I had copied what the README of opentelemetry-collector-releases says, but this seems better.